src/gt4py/backend/numpy_backend.py

```diff
@@ -263,11 +263,6 @@ def _visit_branch_stmt(self, stmt: gt_ir.Statement) -> List[str]:
             target = self.visit(stmt.target)
             value = self.visit(stmt.value)
 
-            data_type = (
-                self.block_info.symbols[target].data_type
-                if target in self.block_info.symbols
-                else stmt.target.data_type
-            )
```

Comment:
Is this a related change?

Reply:
No, this was to satisfy the linter as `data_type` was unused.